lib/iris/experimental/ugrid/mesh.py

```diff
@@ -131,16 +131,16 @@ def __init__(
 
         * indices (numpy.ndarray or numpy.ma.core.MaskedArray or dask.array.Array):
             The index values describing a topological relationship. Constructed
-            of 2 dimensions - the list of locations, and within each location:
-            the indices of the 'target locations' it relates to.
-            Use a :class:`numpy.ma.core.MaskedArray` if :attr:`src_location`
+            of 2 dimensions - the list of 'location elements', and within each:
+            the indices of the 'connected elements' it relates to.
+            Use a :class:`numpy.ma.core.MaskedArray` if :attr:`location_element`
             lengths vary - mask unused index 'slots' within each
-            :attr:`src_location`. Use a :class:`dask.array.Array` to keep
+            :attr:`location_element`. Use a :class:`dask.array.Array` to keep
```

Comment:
As above, 'location element' may be more correct here.

Reply:
https://github.com/SciTools/iris/pull/4498#issuecomment-1014597741